spec/requests/topics_spec.rb

```diff
@@ -0,0 +1,130 @@
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
+  }
+
+  let(:invalid_attributes) {
+    skip("Add a hash of attributes invalid for your model")
+  }
+
+  describe "GET /index" do
+    it "renders a successful response" do
+      Topic.create! valid_attributes
+      get api_v1_topics_url
+      expect(response).to be_successful
+    end
+  end
+
+  describe "GET /show" do
+    it "renders a successful response" do
+      topic = Topic.create! valid_attributes
+      get api_v1_topic_url(topic)
+      expect(response).to be_successful
+    end
+  end
+
+  describe "GET /new" do
+    it "renders a successful response" do
+      get new_api_v1_topic_url
+      expect(response).to be_successful
+    end
+  end
+
+  describe "GET /edit" do
+    it "render a successful response" do
+      topic = Topic.create! valid_attributes
+      get edit_api_v1_topic_url(topic)
+      expect(response).to be_successful
+    end
+  end
+
+  describe "POST /create" do
+    context "with valid parameters" do
+      it "creates a new Topic" do
+        expect {
+          post api_v1_topics_url, params: { topic: valid_attributes }
+        }.to change(Topic, :count).by(1)
+      end
+
+      it "redirects to the created topic" do
+        post api_v1_topics_url, params: { topic: valid_attributes }
+        expect(response).to redirect_to(api_v1_topic_url(Topic.last))
+      end
+    end
+
+    context "with invalid parameters" do
+      it "does not create a new Topic" do
+        expect {
+          post api_v1_topics_url, params: { topic: invalid_attributes }
+        }.to change(Topic, :count).by(0)
+      end
+
+      it "renders a successful response (i.e. to display the 'new' template)" do
+        post api_v1_topics_url, params: { topic: invalid_attributes }
+        expect(response).to be_successful
+      end
+    end
+  end
+
+  describe "PATCH /update" do
+    context "with valid parameters" do
+      let(:new_attributes) {
+        skip("Add a hash of attributes valid for your model")
+      }
+
+      it "updates the requested topic" do
+        topic = Topic.create! valid_attributes
+        patch api_v1_topic_url(topic), params: { topic: new_attributes }
+        topic.reload
+        skip("Add assertions for updated state")
+      end
+
+      it "redirects to the topic" do
+        topic = Topic.create! valid_attributes
+        patch api_v1_topic_url(topic), params: { topic: new_attributes }
+        topic.reload
+        expect(response).to redirect_to(api_v1_topic_url(topic))
+      end
+    end
+
+    context "with invalid parameters" do
+      it "renders a successful response (i.e. to display the 'edit' template)" do
+        topic = Topic.create! valid_attributes
+        patch api_v1_topic_url(topic), params: { topic: invalid_attributes }
+        expect(response).to be_successful
+      end
+    end
+  end
+
+  describe "DELETE /destroy" do
```

Comment:
There should be tests only for the `index` action here, right?

Reply:
yes, that is true, it was the automatic creation from command line